ROADMAP.md

```diff
@@ -0,0 +1,144 @@
+# Roadmap
+
+Welcome to the future!
+
+This page is for features that the Tilt team has concrete
```

Comment:
"concrete plans" makes it sound like we have time frames; is there a way to say "we know about this and have some idea how"?

Reply:
"plaster plans"?

Reply:
Hahahahaha.

"There's a lot we could do; we'd like feedback in how to prioritize these options based on what would be most useful"

Reply:
(anyway, yeah, looks good)